app/models/gear.rb

```diff
@@ -9,14 +9,13 @@
 #  description :string
 #  harm        :integer
 #  armor       :integer
-#  playbook_id :bigint
 #  created_at  :datetime         not null
 #  updated_at  :datetime         not null
 #
-# Gear is equipment that the Hunter can use.
```

Comment:
This is our top-level comment. I think we still want it. We can get rid of the playbook reference though.

Reply:
Resolved with commit 817d7cc.  Tested with the annotate gem and top level documentation was not removed.